site/app/models/Course.php

```diff
@@ -31,6 +31,7 @@ public function __construct(Core $core, $details) {
         $this->semester = $details['semester'];
         $this->title = $details['course'];
         $this->display_name = "";
+        $this->user_group = $details['user_group'];
```

Comment:
Does this cause issues with the rest of the site?

Reply:
Not that I noticed...